src/Examples/UnitModels/Reactors/plug_flow_reactor_testing.ipynb

```diff
@@ -0,0 +1,1147 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "m.fs.thermo_params = GenericParameterBlock(default=thermo_props.config_dict)\n",
+    "m.fs.reaction_params = GenericReactionParameterBlock(default={\"property_package\": m.fs.thermo_params,\n",
+    "                                                              **reaction_props.config_dict})"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Adding Unit Models\n",
+    "\n",
+    "Let us start adding the unit models we have imported to the flowsheet. Here, we are adding a Mixer (assigned a name M101), a Heater (assigned a name H101) and a PFR (assigned a name R101). Note that all unit models need to be given a property package argument. In addition to that, there are several arguments depending on the unit model, please refer to the documentation for more details (https://idaes-pse.readthedocs.io/en/latest/model_libraries/core_lib/unit_models/index.html). For example, the Mixer unit model here is given a `list` consisting of names to the two inlets.\n",
+    "\n",
+    "For the PFR, note that the last three arguments relate to the spacial discretization. They are written explicitly here for demonstration, and are taken as default values (20 elements using backwards finite difference) if not provided - we will only use a single element here for simplicity. The length domain set is assumed to be (0, 1) if not specified otherwise."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "m.fs.M101 = Mixer(default={\"property_package\": m.fs.thermo_params,\n",
+    "                           \"inlet_list\": [\"reagent_feed\", \"catalyst_feed\"]})\n",
+    "m.fs.H101 = Heater(default={\"property_package\": m.fs.thermo_params,\n",
+    "                            \"has_pressure_change\": False,\n",
+    "                            \"has_phase_equilibrium\": False})"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {
+    "scrolled": false
+   },
+   "outputs": [],
+   "source": [
+    "m.fs.R101 = PFR(\n",
+    "            default={\"property_package\": m.fs.thermo_params,\n",
+    "                     \"reaction_package\": m.fs.reaction_params,\n",
+    "                     \"has_heat_of_reaction\": True,\n",
+    "                     \"has_heat_transfer\": True,\n",
+    "                     \"has_pressure_change\": False,\n",
+    "                     \"transformation_method\": \"dae.finite_difference\",\n",
+    "                     \"transformation_scheme\": \"BACKWARD\",\n",
+    "                     \"finite_elements\": 1,\n",
```

Comment:
This is not a good idea - a PFT with one element is just a CSTR. It doesn't cost us much to use more elements, so I would suggest we use a few (5 to 10 maybe).

Reply:
The PFR generates a degree of freedom for each finite element, and the initialization returns an infeasible solution when I fix the heat duty of each element to zero and unfix all other reactor specifications. I'm referring to the [test_pfr](https://github.com/IDAES/idaes-pse/blob/main/idaes/generic_models/unit_models/plug_flow_reactor.py) example as I troubleshoot how to satisfy the degrees of freedom.

Reply:
That indicates that there is a fundamental issue with the formulation of the problem then, which needs to be addressed before we merge this.